webdriver/tests/bidi/browsing_context/context_created/context_created.py

```diff
@@ -104,9 +104,7 @@ async def test_evaluate_window_open_with_url(
     )
 
 
-async def test_navigate_creates_iframes(
-    bidi_session, current_session, top_context, test_page_multiple_frames
-):
+async def test_navigate_creates_iframes(bidi_session, current_session, top_context, test_page_multiple_frames):
```

Comment:
`current_session` is also WebDriver classic. Can we also get rid of it?

Reply:
In `context_created` tests `current_session` is used in `AsyncPoll`, and I'm not sure how to get rid of it for now. Perhaps switching to listening of the `wait_for_event(CONTEXT_CREATED_EVENT)`. As it's not fully implemented yet on our side, I cannot verify if such a change will be valid. So keeping it out of scope.

Reply:
UPD: oh, [`AsyncPoll`](https://github.com/web-platform-tests/wpt/blob/0f90713c40d6a96f9d0f4c92ba36b97c740aa1e3/webdriver/tests/support/sync.py#L27) doesn't care what exactly is in the session. Switched from `current_session` to `bidi_session`.